CHANGES.rst

```diff
@@ -24,14 +24,16 @@ For plans and progress see https://github.com/gammapy/gammapy/milestones/0.4
 Contributors
 ++++++++++++
 
+- Manuel Paz Arribas
 - Axel Donath
 
 Pull requests
 +++++++++++++
 
 - Make background cube models [#319] (Manuel Paz Arribas)
-- Production of true/reco bg cube models should use the same model. [#335] (Manuel Paz Arribas)
-- Fix TS map boundary handling [#332] (Axel Donath) 
+-  Production of true/reco bg cube models should use the same model. [#335] (Manuel Paz Arribas)
+- Fix TS map boundary handling [#332] (Axel Donath)
+- Added example script to produce true and reco bg cube model. [#338] (Manuel Paz Arribas)
```

Comment:
Let's use a uniform format for GH issue titles and changelog. Active voice ("Add" instead of "Added") and no dot at the end of the line.